pkg/executables/kubectl.go

```diff
@@ -1341,11 +1343,23 @@ func (k *Kubectl) GetDaemonSet(ctx context.Context, name, namespace, kubeconfig
 	return obj, nil
 }
 
-func (k *Kubectl) GetResources(ctx context.Context, resourceType string, opts ...KubectlOpt) (string, error) {
+func (k *Kubectl) GetResources(ctx context.Context, resourceType string, opts ...KubectlOpt) (bytes.Buffer, error) {
 	params := []string{
 		"get", resourceType,
 	}
 	applyOpts(&params, opts...)
 	stdOut, err := k.Execute(ctx, params...)
-	return stdOut.String(), err
+	return stdOut, err
+}
+
```

Comment:
I'm not a big fan of having package specific things here. Kubectl shouldn't know what a package is.